latch_cli/main.py

```diff
@@ -912,6 +856,96 @@ def sync(srcs: List[str], dst: str, delete: bool, ignore_unsyncable: bool):
     )
 
 
+"""
+NEXTFLOW COMMANDS
+"""
+
+
+@main.group()
+def nextflow():
+    """Manage nextflow"""
+    pass
+
+
+@nextflow.command("version")
+@click.argument("pkg_root", nargs=1, type=click.Path(exists=True, path_type=Path))
+def version(pkg_root: Path):
+    """Get the Latch version of Nextflow installed for the current project."""
+
+    with open(pkg_root / ".latch" / "nextflow_version", "r") as f:
+        version = f.read().strip()
+
+    click.secho(f"Nextflow version: {version}", fg="green")
+
+
+@nextflow.command("upgrade")
+@click.argument("pkg_root", nargs=1, type=click.Path(exists=True, path_type=Path))
+def generate_entrypoint(
+    pkg_root: Path,
+):
+    """Download the latest version of Nextflow for the current project."""
+
+    from latch_cli.nextflow.dependencies import ensure_nf_dependencies
+
+    ensure_nf_dependencies(pkg_root)
+
+
+@nextflow.command("generate-entrypoint")
+@click.argument("pkg_root", nargs=1, type=click.Path(exists=True, path_type=Path))
+@click.option(
```

Comment:
if someone does `latch nextflow generate-entrypoint .` (without the `--nf-script` option) what happens?